examples/blaze/src/main/scala/com/example/http4s/blaze/BlazeExample.scala

```diff
@@ -23,6 +23,7 @@ import org.http4s.server.{Router, Server}
 import org.http4s.server.blaze.BlazeServerBuilder
 import org.http4s.syntax.kleisli._
 import scala.concurrent.ExecutionContext.global
+import org.http4s.server.syntax.serverBuildable._
```

Comment:
We already have too many imports to get anything done, and this adds another one.